pulp_ansible/app/tasks/collections.py

```diff
@@ -324,7 +324,7 @@ def _get_url(page, api_version):
                 name, version, source = collection_info[page - 1]
                 namespace, name = name.split(".")
                 root = source or remote.url
-                url = f"{root}/api/v2/collections/{namespace}/{name}"
+                url = urljoin(root, f"/api/v{api_version}/collections/{namespace}/{name}")
```

Comment:
it fixes the issue only for galaxy V2, but it won't work for syncing from Galaxy V3, AH or Pulp

Reply:
@bmbouter @daviddavis suggestions?

Reply:
Because those sources use the V3 api?

Reply:
I changed the code, it is enough to solve the situation described on the issue, but I don't like this solution

Reply:
@newswangerd maybe it helps to unblock your tests